cmd/kubeless/deploy.go

```diff
@@ -142,4 +156,6 @@ func init() {
 	deployCmd.Flags().StringP("memory", "", "", "Request amount of memory, which is measured in bytes, for the function. It is expressed as a plain integer or a fixed-point interger with one of these suffies: E, P, T, G, M, K, Ei, Pi, Ti, Gi, Mi, Ki")
 	deployCmd.Flags().Bool("trigger-http", false, "Deploy a http-based function to Kubeless")
 	deployCmd.Flags().StringP("runtime-image", "", "", "Custom runtime image")
+	deployCmd.Flags().Bool("headless", false, "Deploy the function as a headless service")
```

Comment:
The description probably confuses users what exactly `function as a headless service` is. I would rather say: "deploy the function without IP and load balancing support from Kubernetes. See: https://kubernetes.io/docs/concepts/services-networking/service/#headless-services" 

Or something like that.